pkg/proxy/ipvs/netlink_linux.go

```diff
@@ -86,6 +86,16 @@ func (h *netlinkHandle) EnsureDummyDevice(devName string) (bool, error) {
 	return false, h.LinkAdd(dummy)
 }
 
+// IsDeviceExist is part of interface
+func (h *netlinkHandle) IsDeviceExist(devName string) (bool, error) {
+	_, err := h.LinkByName(devName)
+	if err == nil {
+		// found dummy device
+		return true, nil
+	}
+	return false, fmt.Errorf("device %s do not exist", devName)
```

Comment:
I would suggest return error directly instead of eating it.

Reply:
make sense, thanks